tools/go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/golangci/golangci-lint v1.35.2
 	github.com/hashicorp/go-changelog v0.0.0-20201005170154-56335215ce3a
-	github.com/katbyte/terrafmt v0.2.1-0.20200913185704-5ff4421407b4
+	github.com/katbyte/terrafmt v0.3.0
 	github.com/pavius/impi v0.0.3 // indirect
 	github.com/terraform-linters/tflint v0.20.3
 )
 
-replace github.com/katbyte/terrafmt => github.com/gdavison/terrafmt v0.2.1-0.20201026181004-a896893cd6af
+replace github.com/katbyte/terrafmt => github.com/gdavison/terrafmt v0.3.1-0.20210204054728-84242796be99
```

Comment:
Is it possible to get off this fork yet?

Reply:
Soon, hopefully. It's using an actual Go parser instead of string matching to extract the Terraform blocks.

Reply:
Okay -- for whatever reason dependabot isn't happy with this type of Go module replacement setup.